xml/docu_styleguide.language.xml

```diff
@@ -76,6 +76,32 @@
   </sect3>
  </sect2>
 
+ <sect2 xml:id="sec-bias">
+  <title>Biases and Inclusivity</title>
+  <para>
+   Do not artificially limit your audience by excluding or offending members of
+   it.
+  </para>
+  <para>
+   Avoid indicating gender in your documentation.
+   If possible, use plural to allow use of <quote>they</quote> as the pronoun.
+   Otherwise, use <quote>he or she.</quote>
+  </para>
+  <para>
+   &suse; supports the Inclusive Naming Initiative which aims to help avoid
+   harmful language.
+   When making language choices for documentation, review the Initiative's
```

Comment:
lowercase 'initiative', it's not a name here.
```suggestion
   When making language choices for documentation, review the initiative's
```

Reply:
Splitting hairs imo :) My intention was to shorten the organization name on second mention, in that sense, my version is just as correct as yours, the meaning is just a teensy bit different.